tests/phpunit/FileSystem/SourceFileCollectorTest.php

```diff
@@ -109,107 +102,9 @@ public function sourceFilesProvider(): iterable
             ],
         ];
 
-        yield 'filter by file name, no excludes' => [
```

Comment:
We're delegating it to `FilteredEnrichedTraceProvider` but it feels like the later is not as extensively tested as the previous version of `SourceFileCollector`... Should we add back those tests to `FilteredEnrichedTraceProvider`?

Reply:
`RealPathFilterIteratorTest` has these covered already IIRC

Reply:
Anyway, there isn't any apparent loss of coverage or MSI degradation.

Reply:
And, yes, `SourceFileCollectorTest` has a fairly extensive list of fixtures, which look somewhat similar to what we have removed here.

Reply:
👍